### DISCUSSION TITLE
Click an image at top right corner of the page

### DISCUSSION THREAD

**Discussion by SevenOutman** [getgauge/taiko#2562] [General]
Hi, thanks for this great tool!

I have this use case where an image as user's avatar is placed at the top right corner of the page (like in this GitHub page). I want to click that image to open its popup menu but didn't find a selector for it.
<img width="368" alt="image" src="https://user-images.githubusercontent.com/8225666/159840862-0b400ead-dbfb-4f91-96a8-1aad420986b1.png">

**Comment by gulshankamboj** [comment]
https://docs.taiko.dev/api/image/ 

<img width="753" alt="Screenshot 2022-04-28 at 9 53 26 PM" src="https://user-images.githubusercontent.com/82322100/165799279-b97db80c-a538-4d5e-aa36-9081992bd05c.png">

Try this image api and if it doesn't work try using the xpath:

click($("xpath-here"))

For Github xpath is like, "//summary[@class='Header-link']/img"

So use, click($("//summary[@class='Header-link']/img"))

Let me know if it works!

**Reply by SevenOutman** [reply]
Thanks for the answer. But the ideal query should not be related to the implementation detail IMO, which means to be semantic and to avoid using XPath or CSS selectors. That's also what I believe taiko is trying to achieve.

I was expecting something like this:

```js 
image(atTopRightCorner)
```
